src/Eccube/Service/MailService.php

```diff
@@ -440,4 +375,24 @@ public function sendPasswordResetCompleteMail(\Eccube\Entity\Customer $Customer,
         return $count;
     }
 
+    /**
+     * @param string $mailTo
+     * @param string $subject
+     * @param string $body
+     * @return \Swift_Mime_MimePart
+     */
+    private function createMailMessage($mailTo, $subject, $body)
```

Comment:
please dont make method.( dont refactor)
Because We have plan to refactor large-scale sourcecode about email function.